common/src/camera/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# header only currently
+
+#set(NBL_LIB_SOURCES
+#    "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp"
+#)
+
+#nbl_create_ext_library_project(Camera "" "${NBL_LIB_SOURCES}" "" "" "")
```

Comment:
why do we have a separate lib for cameras?

Reply:
we don't `CCamera` is header only, going to nuke this

Reply:
nuke plz

Reply:
why not nuked yet?

Reply:
bump